docs/concepts/operator.md

```diff
@@ -232,6 +232,106 @@ Please refer to [this document](../usage/shoot_credentials_rotation.md#gardener-
 
 ⚠️ Rotation of static `ServiceAccount` secrets is not supported since the `kube-controller-manager` does not enable the `serviceaccount-token` controller.
 
+## Upgrading An Existing Gardener Landscape To `gardener-operator`
+
+Since `gardener-operator` was only developed in 2023, six years after the Gardener project initiation, most users probably already have an existing Gardener landscape.
+The most prominent installation procedure is [garden-setup](https://github.com/gardener/garden-setup), however experience shows that most community members have developed their own tooling for managing the garden cluster and the Gardener control plane components.
+
+> Consequently, providing a general upgrade guide is not possible since the detailed steps vary heavily based on how the components were set up previously.
+> As a result, this section can only highlight the most important caveats and things to know, while the concrete upgrade steps must be figured out individually based on the existing installation.
+
+Please make sure that you configure all your desired fields in the [`Garden` resource](#garden-resources).
+
+### ETCD
+
+`gardener-operator` leverages `etcd-druid` for managing the `virtual-garden-etcd-main` and `virtual-garden-etcd-events`, similar to what happens for shoot clusters.
+The `PersistentVolumeClaim` names differ slightly - for `virtual-garden-etcd-events` it's `virtual-garden-etcd-events-virtual-garden-etcd-events-0`, while for `virtual-garden-etcd-main` it's `main-virtual-garden-etcd-virtual-garden-etcd-main-0`.
+
+The backup bucket must be created separately, and its name as well as the respective credentials must be provided via the `Garden` resource in `.spec.virtualCluster.etcd.main.backup`.
+
```

Comment:
So what is the ask here? Should an operator create a corresponding `PersistentVolumeClaim` and moving all existing data into the volume before starting the operator?
Same question about the backup.

In other words, do we want to provide more actionable documentation or leave the action items in the fineprint?